app/cdap/components/Connections/Browser/GenericBrowser/index.tsx

```diff
@@ -172,7 +169,7 @@ export function GenericBrowser({ selectedConnection }) {
         schema,
       });
     } catch (e) {
-      setError(`Failed to get plugin information. Error: ${getErrorMessage(e)}`);
+      setError(`Failed to get plugin information. Error: ${getApiErrorMessage(e)}`);
```

Comment:
does this e come from a different place? getPluginSpec is an api function that calls into the code you modified in apiHelpers.ts, right?

Reply:
Ah I see that I misread how general the modification in apihelpers was